deps.edn

```diff
@@ -32,7 +32,7 @@
   ;; - HugSql: Use custom version instead of the released version (0.5.1)
   com.stuartsierra/component               {:mvn/version "1.0.0"}
   com.github.seancorfield/next.jdbc        {:mvn/version "1.2.709"}
-  com.yetanalytics/hugsql                  {:mvn/version "0.6.0-SNAPSHOT"}
+  com.yetanalytics/hugsql                  {:mvn/version "0.6.1-SNAPSHOT"}
```

Comment:
Fragments, which were not included in the update.

Reply:
Gotcha. One thing here, since we offer this out, `SNAPSHOT` versions are probably not great to put in a release, as they are replaceable by convention. I'd advise cutting a release of our fork.

Reply:
`lein` used to yell at you about it, though I think all of the deps/cli build tooling is indifferent

Reply:
What do you suggest as a release name? `0.6.1` would be the obvious choice here, but it may get iffy once the main branch gets updated to that version.

Reply:
Maybe another qualifier that isn't `SNAPSHOT`? I really don't know what the convention is for forks

Reply:
but given that the group id is different, it probably doesn't matter.. Does a fork get parent tags I wonder?

Reply:
Yeah `0.5.3` would probably be simplest

Reply:
Some discussion here: https://github.com/semver/semver/issues/17

Reply:
Since we don't have to strictly follow SemVer I think something like `0.5.3-FORK` would be fine (and I'm downgrading the version since I don't think a v0.6.1 with fragments will come out anytime soon anymore).

Reply:
So v + v? that works I guess

Reply:
though you might need to modify your regex in gh actions ;)

Reply:
> Since we don't have to strictly follow SemVer I think something like `0.5.3-YetAnalytics` would be fine (and I'm downgrading the version since I don't think a v0.6.1 with fragments will come out anytime soon anymore).

I kind of liked the convention they discuss near the end of that issue, where it's like v<parent version>+<branch version>, remember that yetanalytics is already in the group ID

Reply:
https://github.com/yetanalytics/lrsql/pull/219/commits/a37fd2409a8eb8776b2e374a2407998083bf0f1a